lib/global.gi

```diff
@@ -97,6 +97,21 @@ InstallGlobalFunction( IsBoundGlobal,
 end);
 
 
+#############################################################################
+##
+#M  IsAutoGlobal ( <name> )  . . . .  check if a global is automatic
+## 
+
+
+InstallGlobalFunction( IsAutoGlobal, 
+        function (name)
+    local isauto;
+    CheckGlobalName( name );
```

Comment:
Is there are benefit to calling `CheckGlobalName` here?

Reply:
For consistency with IsBoundGlobal